doc/tutorials/dune-package-management/repos.md

```diff
@@ -0,0 +1,50 @@
+## Custom Repositories
+
+In normal usage Dune looks up packages from two sources:
+
+  1. The upstream, community maintained opam repository at
+     [ocaml/opam-repository](https://github.com/ocaml/opam-repository) for most
+     packages
+  2. An overlay opam repository with patched software to allow usage in Dune at
+     [ocaml-dune/opam-overlays](https://github.com/ocaml-dune/opam-overlays)
+
+This list is built-in but can be configured. To change the repositories to be
+used the `dune-workspace` file has to be edited (and created if it didn't
+exist):
```

Comment:
```suggestion
This list is built-in but can be configured. To change the repositories, the `dune-workspace` file has to be edited (and created if it didn't
exist):
```

is "to be used" necessary here? Could it be inferred that the repositories they need to change are the ones they want to use?

Reply:
I think it isn't necessary. I mostly added it because configuration consists of adding them and then configuring them to be used, but people will surely figure that out.